lib/crypto/test/crypto_SUITE.erl

```diff
@@ -1186,6 +1189,13 @@ exor(Config) when is_list(Config) ->
     do_exor(<<1, 2, 3, 4, 5, 6, 7, 8, 9, 10>>),
     do_exor(term_to_binary(lists:seq(1, 1000000))).
 %%--------------------------------------------------------------------
+hash_equals() ->
+    [{doc, "Test the hash_equals function"}].
+hash_equals(Config) when is_list(Config) ->
+    true = crypto:hash_equals(<<>>, <<>>),
+    true = crypto:hash_equals(<<"abc">>, <<"abc">>),
+    false = crypto:hash_equals(<<"abc">>, <<"abe">>).
```

Comment:
To avoid a failed test in case of missing CRYPTO_memcmp one can change the test into something like:
```
try
       true = crypto:hash_equals(<<>>, <<>>),
￼       true = crypto:hash_equals(<<"abc">>, <<"abc">>),
￼       false = crypto:hash_equals(<<"abc">>, <<"abe">>)
catch
       error: {notsup,{"hash_equals.c",_Line},"Unsupported CRYPTO_memcmp"} -> {skip, "No CRYPTO_memcmp"}
end

```

Reply:
Fixed and I've squashed all the commits.